src/main.js

```diff
@@ -1,4 +1,5 @@
 import app from './app'
 import './themes'
+import './icons'
```

Comment:
Is importing icons inside main really the best we can do here?

I do not want to go to direction where we'll add imports inside `main.js` but rather inside app.

Reply:
I messed up here -- it completely slipped my mind, that it is not required to add all used FA icons on initialization, but can be added later on.
Because of that, I moved this import outside of app, since existing icons would be re added.

The mistake is now resolved & code is refactored.